examples/app/sliders.py

```diff
@@ -1,17 +1,24 @@
 ''' Present an interactive function explorer with slider widgets.
 
-Scrub the sliders to change the properties of the ``sin`` curve, or
-type into the title text box to update the title of the plot.
+.. note::
```

Comment:
I think it's better if notes are kept to 1-2 simple sentences. Since this has code blocks I think the `.. note::` should be removed and the block be unindented